content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -14,8 +14,7 @@ descriptionLong: |
   projects by our users and big ideas about best practices in ML and data
   science.
 picture: 2020-04-06/april_header.png
-pictureComment:
-  A view from [Barrancas del
+pictureComment: A view from [Barrancas del
   Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by Jorge Orpinel
```

Comment:
BTW I guess this should be manually reformatted so the link isn't broken into 2 lines. Let's see if it lets me...

Reply:
It does 🙂 (3a27d70a)